src/components/review-list.jsx

```diff
@@ -20,16 +20,22 @@ export default function ReviewList () {
             setIsLoading(false)
         })
     }, [])
+    
+    let navigate = useNavigate()
+    function viewReview (review_id) {
+        setIsLoading(true);
+        navigate(`/reviews/:${review_id}`)
+    }
 
     return (
         <>
         {isLoading ? (<p>Loading...</p>) : (
             reviews.map((review) => {
-               return <div key={review.review_id} className="preview_review_card">
+               return <div key={review.review_id} className="preview-review-card">
                    <p><b>{review.title}</b></p>
                    <img src={review.review_img_url} alt={review.title} width="100"></img>
                    <p>{review.review_body.slice(0, 71)}...</p>
-                   <button type="button">Read</button>
+                   <button onClick={() => {viewReview(review.review_id)}} type="button">Read 👀</button>
```

Comment:
You could use a Link tag here, I think inside a button it has to be wrapped around the text part only? I feel like you may have tried it then resorted to useNavigate instead. To be honest it works either way so upto you!